components/dashboard/src/start/CreateWorkspace.tsx

```diff
@@ -129,15 +129,15 @@ export class CreateWorkspace extends React.Component<CreateWorkspaceProps, Creat
     }
 
     else if (result?.runningWorkspacePrebuild) {
-      statusMessage = <p className="text-base text-gray-400">⚡Prebuild in progress</p>;
-      logsView = <Suspense fallback={<div className="m-6 p-4 h-60 w-11/12 lg:w-3/5 flex-shrink-0 rounded-lg" style={{ color: '#8E8787', background: '#ECE7E5' }}>Loading...</div>}>
-        <WorkspaceLogs />
-      </Suspense>;
+      return <RunningPrebuildView
+        runningPrebuild={result.runningWorkspacePrebuild}
+        onIgnorePrebuild={() => this.createWorkspace(CreateWorkspaceMode.ForceNew)}
+        onPrebuildSucceeded={() => this.createWorkspace(CreateWorkspaceMode.UsePrebuild)}
+      />;
     }
 
     return <StartPage phase={phase} error={!!error}>
       {statusMessage}
-      {logsView}
       {error && <div>
         <a href={gitpodHostUrl.asDashboard().toString()}><button className="mt-8 px-4 py-2 text-gray-500 bg-white font-semibold border-gray-500">Go back to dashboard</button></a>
```

Comment:
**suggestion**: Since we're here, could we also add a hover state for this one? 🏓 

```suggestion
        <a href={gitpodHostUrl.asDashboard().toString()}><button className="mt-8 px-4 py-2 text-gray-500 bg-white font-semibold border-gray-500 hover:bg-gray-100 hover:border-gray-700 hover:text-gray-700">Go back to dashboard</button></a>
```